aspnetcore/blazor/class-libraries.md

```diff
@@ -106,6 +106,42 @@ Welcome to your new app.
 
 Include the `@using MyComponentLib1` directive in the top-level *_Import.razor* file to make the library's components available to an entire project. Add the directive to an *_Import.razor* file at any level to apply the namespace to a single page or set of pages within a folder.
 
+## Create a Razor components class library with static assets
+
+An RCL can include static assets. The static assets are available to any app that consumes the library. For more information, see <xref:razor-pages/ui-class#create-an-rcl-with-static-assets>.
+
+## Add an XML linker configuration file to a library
```

Comment:
This content looks good, but it feels a bit out of context because all of the content explaining what the linker does and why a config file might be necessary lives in the "Configure the linker" doc. I know we discussed putting this content here in the component libraries doc, but technically this content isn't specific to component libraries. It applies to *any* library that you might be using in a Blazor WebAssembly app. I think we should move this section to the end of the "Configure the linker" doc. 

Also, the "Configure the linker" doc points to https://github.com/mono/linker/blob/master/src/linker/README.md#syntax-of-xml-descriptor for the syntax of the XML file, but there is a much more extensive syntax description that I think we should link to instead here: https://github.com/mono/linker#link-xml-file-examples.